tools/release/release.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  releaseChangelog,
+  releasePublish,
+  releaseVersion,
+} from 'nx/src/command-line/release';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import yargs from 'yargs';
+
+void (async () => {
+  try {
+    const options = await yargs
+      .version(false)
```

Comment:
[Bug] On my machine's installed version at least, `yargs` needs to be provided with the argv now:

```suggestion
    const options = await yargs(process.argv.slice(2))
      .version(false)
```

Reply:
How strange… I tested it multiple times on my machine without issue, and I also used it like this in other repos

Reply:
Should be handled by the explicit yargs version being specified in the package.json now